heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -235,6 +235,13 @@
   public static final String TOPOLOGY_UPDATE_REACTIVATE_WAIT_SECS =
       "topology.update.reactivate.wait.secs";
 
+  /**
+   * Topology-specific environment variables for the worker child process.
```

Comment:
There is no notion of worker child process in heron - can you rephrase it?

Reply:
will fix